streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsPartitionAssignor.java

```diff
@@ -793,6 +811,14 @@ private boolean assignTasksToClients(final Cluster fullMetadata,
         return probingRebalanceNeeded;
     }
 
+    private org.apache.kafka.streams.processor.assignment.TaskAssignor createUserTaskAssignor(final boolean lagComputationSuccessful) {
+        if (!lagComputationSuccessful) {
+            log.info("Failed to fetch end offsets for changelogs, will return previous assignment to clients and "
+                     + "trigger another rebalance to retry.");
+        }
```

Comment:
seems weird to make this part of this function? is this relevant only in the case when we're creating a user assignor?